src/libraries/System.Private.CoreLib/src/System/IO/RandomAccess.Windows.cs

```diff
@@ -793,23 +793,22 @@ internal unsafe void FreeNativeOverlapped(NativeOverlapped* pOverlapped)
         }
 
         // Abstracts away the type signature incompatibility between Memory and ReadOnlyMemory.
-        // TODO: Use abstract static methods when they become stable.
         private interface IMemoryHandler<T>
         {
-            int GetLength(in T memory);
```

Comment:
I removed the `in`; everywhere I know we pass memories by value.

Reply:
We tend to avoid `in` for public APIs since the tradeoffs are complex.

It is fine to use it for internal implementation details like this one where the tradeoffs can be fully understood.

Reply:
`in` provides a small code size benefit in this case.

Reply:
I brought the `in` back.